### ISSUE TITLE
TIT3 tag - "version" or "subtitle?"

### ISSUE THREAD

**Issue by Beckfield** [puddletag/puddletag#573]
I recently acquired a Windows system, and have installed MP3Tag on it.  I was surprised when I used it to open some music files whose tags were written by Puddletag.

In Puddletag, I entered text in the TIT3 frame (which Puddletag calls 'version').  
When MP3Tag opens these files, the TIT3 frame (which MP3Tag callse 'subtitle') is blank.  As an experiment, I populated the 'subtitle' frame with different text than the 'version' frame, saved it, and refreshed the files in Puddletag.  Sure enough, the TIT3 frame retained the original, Puddletag-written text, and the MP3Tag text is not shown.

Viewing one of the files using a Hex editor, I found a 'version' frame and a 'subtitle' frame embedded in the file.

So Puddletag and MP3Tag disagree on this particular tag.  I did a little research, and I found that Puddletag seems to be the 'odd man out.'  I tried a couple other tag editors, and they all found the MP3Tag-written frame, but not the Puddletag-written one.  The ID3 spec calls TIT3 'subtitle/description refinement.'

**Comment by sandrotosi** [comment]
indeed: https://github.com/puddletag/puddletag/blob/master/source/puddlestuff/audioinfo/id3.py#L163

it may be a good idea to review all of those fields
